back/node_watcher/src/nodeWatcher.ts

```diff
@@ -116,12 +116,8 @@ async function incrementor(
 
       const result = await task.read(blockHash, cached, api);
 
-      try {
-        l.warn(`Writing: ${JSON.stringify(result)}`);
-        await task.write(blockNumber, result);
-      } catch (e) {
-        l.error(e);
-      }
+      l.warn(`Writing: ${JSON.stringify(result)}`);
+      await task.write(blockNumber, result);
```

Comment:
We were ignoring writing to Prisma errors before, hence skipping them, I'm not sure why. Now, if there's an error while writing to Prisma, we throw

Reply:
I wonder if this will throw for errors such as unique constraints violated. I guess this is fine for the tip of the chain, but any resync with a modified schema would fail.
We can try still.

Reply:
Ah right. Let's do small steps then, I reverted this.